api/handlers.go

```diff
@@ -359,6 +383,18 @@ func getVolumes(c *context, w http.ResponseWriter, r *http.Request) {
 			continue
 		}
 
+		found = false
+		for _, driver := range drivers {
+			if volume.Driver == driver {
```

Comment:
Is there any reason you can't use `filters.ExactMatch`, just like the daemon code?

Reply:
no reason, I was just following the existing pattern in `name` and `node` matching, but yes, using ExactMatch is much cleaner. 
shall I change the name/node filter matching to mirror the daemon? ie using `filters.Match`?
or "if it ain't broke don't fix it"

Reply:
Yeah, let's leave the name/node filtering alone. Let's keep this PR focused.

Reply:
sounds good! 
PR is updated with ExactMatch